src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -124,19 +123,15 @@ public static LocalDateTime getFirstDateTimeMustBe() {
     }
 
     public static int getHearingDuration(SscsCaseData caseData, ReferenceDataServiceHolder referenceDataServiceHolder) {
-        // TODO Adjournments - Check this is the correct logic for Adjournments
-        // TODO Future Work - Manual Override
-
         Integer duration = getHearingDurationAdjournment(caseData);
         if (isNull(duration)) {
             Integer overrideDuration = Optional.ofNullable(caseData.getSchedulingAndListingFields())
-                .map(SchedulingAndListingFields::getOverrideSchedulingListingFields)
-                .map(OverrideSchedulingListingFields::getOverrideDuration)
-                .orElse(0);
+                .map(SchedulingAndListingFields::getOverrideDuration)
+                .orElse(null);
 
-            duration = overrideDuration > 0
-                ? overrideDuration
-                : getHearingDurationBenefitIssueCodes(caseData, referenceDataServiceHolder);
+            duration = overrideDuration == null || overrideDuration == 0
```

Comment:
Unless the check is being done on the ExUI side, I would rather check the override duration is greater than 0 (or in this new change, less than 1). Otherwise, people could add negative durations and then HMC would reject the call.